### ISSUE TITLE
Volvo XC40 2024: access popup always shown at car start

### ISSUE THREAD

**Issue by tibirna** [MoleMan1024/audiowagon#152]
**Describe the issue**
USB access popup is always shown at car start, even when checking the box to "Always open AudioWagon when drive XXXX is connected".

This is compounded by the fact that the popup is sometimes shown several seconds _after_ the car started moving, which prompts the OS to block the interaction with the screen because "action is dangerous during driving" (stupid, but this is what this OS is programmed to do...). Following that, the drive is not accessible anymore, even if I stop the car to allow for the interaction with the access granting popup to go on and then select "OK". 

Afterwards, the only way to get the application to access the USB drive is to stop the car, disconnect the USB drive, reconnect it, wait several tens of seconds for the access popup  to appear, touch "OK" and wait for the scanning to complete.

**Expected behavior**
The application should request granting access to the USB drive only once across multiple car starts, as long as the drive remains connected. The access granting dialog should be configured to avoid blocking by the OS (as it only requires selecting "OK" or "Cancel", and doesn't require typing or other complex operations).

**AudioWagon version**
Version published on 2024-07-07.

**Car version**
Volvo Google Automotive 3.1.9

**Comment by micksmothers** [comment]
Just FYI, this is the first item addressed in the [FAQ](https://moleman1024.github.io/audiowagon/faq.html) and is also already called out in at least one closed issue. I'm sure MoleMan will jump in and add, but this can't be addressed except for by the Android Auto team at Volvo/Polestar/etc. as far as I can tell. It's been there since I first started using AudioWagon over 3 years ago. I will say that the prompt masking while in motion just appeared in the 3.x version of AA, but I just loaded 3.1.9 and I keep forgetting to test if it's back to being accessible in motion as I've gotten in the habit of waiting to move until it's popped up and I've dismissed the dialog.
